src/main/java/uk/gov/hmcts/darts/event/helper/DarNotifyAsyncHelper.java

```diff
@@ -53,9 +53,11 @@ public void notifyDarPcAsync(DarNotifyApplicationEvent event, List<String> openC
 
         try {
             dartsGatewayClient.darNotify(darNotifyEvent);
-            log.info("DarNotify request sent successfully: event_id={}, notification_url={}", event.getDartsEvent().getEventId(), notificationUrl);
+            log.info("DarNotify request sent successfully: event_id={}, notification_url={}, event_type={}",
```

Comment:
I wasn't 100% sure if this was being used by Dynatrace or not. I've got a separate PR to improve the logs here anyway, so maybe remove from this PR? My PR for reference: https://github.com/hmcts/darts-api/pull/2963